test/linear.spec.ts

```diff
@@ -325,18 +252,37 @@ describe('linear pool tests', () => {
                     maxPools
                 );
 
-                assert.equal(paths.length, 2);
+                assert.equal(paths.length, 1);
+
                 checkPath(
-                    ['linearUSDT', 'multiid', 'linearUSDC', 'weightedUsdcWeth'],
+                    ['weightedBalStaBal3', 'staBal3Id', 'linearDAI'],
                     poolsAllDict,
                     paths[0],
                     tokenIn,
                     tokenOut
                 );
+            });
+
+            it('should return 2 valid linear paths', async () => {
+                const tokenIn = USDC.address;
+                const tokenOut = BAL.address;
+                const maxPools = 10;
+                const testPools: any = cloneDeep(smallLinear.pools);
+
+                const [paths, poolsAllDict] = getPaths(
+                    tokenIn,
+                    tokenOut,
+                    SwapTypes.SwapExactIn,
+                    testPools,
+                    maxPools
+                );
+
+                assert.equal(paths.length, 1);
+                // TokenIn>[WeightedPool]>staBal3Bpt>[staBAL3]>bStable>[LINEARPOOL]>TokenOut
```

Comment:
This commented path doesn't correspond to the path below

Reply:
Corrected - thanks.